tests/e2e_test/kyma_test.go

```diff
@@ -0,0 +1,106 @@
+//go:build kyma_e2e
```

Comment:
I suggest integrating this e2e scenario into `deletion_e2e`, the scenario of this test is more suitable for "deleting"